### PR TITLE
GT-1386 add shared request operation package

### PR DESCRIPTION
**Pull request by levieggertcru**
Removed all the RequestOperation classes and generic json decoding logic into the shared request-operation-ios swift package.

### CODE REVIEW

File: godtools/App/Services/Articles/Repository/Downloader/ArticleAemDownloadOperation.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import RequestOperation
```
Comment (gyasistory) [inline godtools/App/Services/Articles/Repository/Downloader/ArticleAemDownloadOperation.swift:10]: @levieggertcru why do you need the import if now code is added?

File: godtools/App/Services/Articles/Repository/Downloader/ArticleAemDownloader.swift [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 import RealmSwift
+import RequestOperation
```
Comment (gyasistory) [inline godtools/App/Services/Articles/Repository/Downloader/ArticleAemDownloader.swift:11]: @levieggertcru same question here?

File: godtools/App/Services/EmailSignUp/EmailSignUpService.swift [hunk 3]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import RequestOperation
```
Comment (gyasistory) [inline godtools/App/Services/EmailSignUp/EmailSignUpService.swift:10]: @levieggertcru again here

**Review by reldredge71 — APPROVED** [review]

File: godtools/App/Share/ViewModels/TranslationDownloaderErrorViewModel.swift [hunk 4]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import RequestOperation
```
Comment (gyasistory) [inline godtools/App/Share/ViewModels/TranslationDownloaderErrorViewModel.swift:10]: here also

**Review by gyasistory — CHANGES_REQUESTED** [review]
Not sure if the imports with no changes are needed.

File: godtools/App/Services/Articles/Repository/Downloader/ArticleAemDownloadOperation.swift (hunk 1 above)
Comment (levieggertcru) [reply godtools/App/Services/Articles/Repository/Downloader/ArticleAemDownloadOperation.swift:10]: Hey @gyasistory that code is now in a shared framework (pulled in via swift package manager) here -> https://github.com/CruGlobal/request-operation-ios.

**Pull request merged**